odoo/addons/base/models/res_users.py

```diff
@@ -1243,6 +1243,13 @@ def _update_user_groups_view(self):
             new_context.pop('install_filename', None)  # don't set arch_fs for this computed view
             new_context['lang'] = None
             view.with_context(new_context).write({'arch': xml_content})
+    
+    def _authorisation_for_user_type_fields(self):
+        """
+        Group that can view the user type. This has been put in a
+        method so it can overwritten.
```

Comment:
Super nit: "overridden" instead of "overwritten"?

Reply:
yep changed :slightly_smiling_face: